src/schema/datatypes/eeg.yaml

```
@@ -11,8 +11,56 @@
     - .fdt
     - .bdf
   entities:
-    sub: required
-    ses: optional
+    subject: required
+    session: optional
     task: required
-    acq: optional
+    acquisition: optional
     run: optional
+- suffixes:
+    - channels
+  extensions:
+    - .json
+    - .tsv
+  entities:
+    subject: required
+    session: optional
+    task: required
+    acquisition: optional
+    run: optional
+- suffixes:
+    - coordsystem
+  extensions:
+    - .json
+  entities:
+    subject: required
+    session: optional
+    acquisition: optional
+- suffixes:
+    - electrodes
+  extensions:
+    # No json, since electrodes.tsv uses coordsystem.json for metadata
```

Actually, JSON is permitted, I think I was wrong about this in the past

```suggestion
     - .json
```